blog.html

```diff
@@ -0,0 +1,204 @@
+          
+          <div class="blog-post__meta glass-panel">
+            <p class="blog-post__author">
+              <strong>Tony Nguyen</strong>
+              <time class="blog-post__date" datetime="2022-06-20">20 Jun 2022</time>
+            </p>
+          </div>
+        </div>
+
+        <div class="blog-post__content glass-panel">
+          <ul class="blog-post__description">
+            <li>
+              <p>
+                The Basilisks that <a href="#">players</a> collect are represented
+                with NFTs on Ethereum's blockchain with real-world value. Players
+                use the <a href="#">Basilisks they</a> have gathered to battle
+                other players to win ether (ETH).
+              </p>
+            </li>
+            <li>
+              <p>
+                In Balthazar Dragons, players are immersed in a 3D open world to
+                explore and capture dragon-like beasts called Basilisks. Balthazar
+                Dragons is an upcoming fantasy role-playing game developed on the
+                Ethereum blockchain by a decentralized autonomous organization
+                (DAO) called the Balthazar DAO.
+              </p>
+            </li>
+          </ul>
+        </div>
+
+        <blockquote class="blog-post__quote glass-panel">
+          <p>
+            "The first huge metaverse arena to arrange live sports, hang out and
+            even make bets and wagers"
+          </p>
+          <span class="blog-post__rating">⭐ 8.5/10 Markout Corporation</span>
+        </blockquote>
+
+        <div class="blog-post__gallery glass-grid">
+          <figure>
+            <img src="basilisk-image-2.jpg" alt="Basilisk Image 2" />
+            <figcaption>Second view of the Basilisk creature</figcaption>
+          </figure>
+          <figure>
+            <img src="basilisk-image-3.jpg" alt="Basilisk Image 3" />
+            <figcaption>Third view of the Basilisk creature</figcaption>
+          </figure>
+        </div>
+
+        <div class="blog-post__footer glass-panel">
+          <div class="blog-post__tags">
+            <span>Tags: </span>
+            <p>NFT</p>
+            <a href="#" class="glass-tag">NFT</a>
+            <a href="#" class="glass-tag">ICO</a>
+            <a href="#" class="glass-tag">Crypto</a>
+          </div>
+
+          <div class="blog-post__share glass-panel">
+            <p>Share: </p>
+            <div class="blog-post__social-links">
+              <a href="#" class="glass-icon"><img src="twitter.svg" alt="Twitter" /></a>
+              <a href="#" class="glass-icon"><img src="facebook.svg" alt="Facebook" /></a>
+              <a href="#" class="glass-icon"><img src="telegram.svg" alt="Telegram" /></a>
+            </div>
+          </div>
+        </div>
+      </article>
+    </main>
+
+    <footer class="footer">
+      <div class="logo footer__logo"></div>
+        <a href="index.html" class="footer__logo-link">

```

Comment:
Link and images it's a element of `logo` block, not `footer` block.

Change it in `header `too.